docs/source/conf.py

```diff
@@ -115,29 +115,20 @@
 # a list of builtin themes.
 #
 # html_theme = 'sphinx_rtd_theme'
-html_theme = 'pydata_sphinx_theme'
+# html_theme = 'pydata_sphinx_theme'
 # html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 # zerovm-sphinx-theme
 
+html_theme = 'pyansys_sphinx_theme'
 html_logo = '_static/pyansys-logo-black-cropped.png'
-
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-#
 html_theme_options = {
-    "github_url": "https://github.com/pyansys/DPF-Core",
-    # "show_toc_level": 1,
-    # "show_prev_next": False
```

Comment:
You might consider keeping `"show_prev_next": False` set.  Docs look better without it.